ambry-frontend/src/main/java/com/github/ambry/frontend/FrontendMetrics.java

```diff
@@ -269,6 +285,25 @@ public FrontendMetrics(MetricRegistry metricRegistry) {
     undeleteBlobSecurityProcessResponseMetrics =
         new AsyncOperationTracker.Metrics(UndeleteHandler.class, "securityProcessResponse", metricRegistry);
 
+    putSecurityProcessRequestMetrics =
+        new AsyncOperationTracker.Metrics(PutBlobHandler.class, "putSecurityProcessRequest", metricRegistry);
+    putSecurityPostProcessRequestMetrics =
+        new AsyncOperationTracker.Metrics(PutBlobHandler.class, "putSecurityPostProcessRequest", metricRegistry);
+    putReadStitchRequestMetrics =
+        new AsyncOperationTracker.Metrics(PutBlobHandler.class, "putReadStitchRequest", metricRegistry);
+    putRouterStitchBlobMetrics =
+        new AsyncOperationTracker.Metrics(PutBlobHandler.class, "putRouterStitchBlob", metricRegistry);
+    putRouterPutBlobMetrics =
+        new AsyncOperationTracker.Metrics(PutBlobHandler.class, "putRouterPutBlob", metricRegistry);
+    putIdConversionMetrics =
+        new AsyncOperationTracker.Metrics(PutBlobHandler.class, "putIdConversion", metricRegistry);
+    putSecurityProcessResponseMetrics =
+        new AsyncOperationTracker.Metrics(PutBlobHandler.class, "putSecurityProcessResponse", metricRegistry);
+    putBlobRouterMetrics =
+        new AsyncOperationTracker.Metrics(PutBlobHandler.class, "router", metricRegistry);
+    putBlobSecurityProcessResponseMetrics =
+        new AsyncOperationTracker.Metrics(PutBlobHandler.class, "securityProcessResponse", metricRegistry);
```

Comment:
is  this a duplicate of `putSecurityProcessResponse`?

Reply:
Thanks for catching that. Removed the duplicate one.